src/workerd/api/node/BUILD.bazel

```diff
@@ -14,7 +14,6 @@ wd_cc_library(
         "@capnp-cpp//src/kj/compat:kj-gzip",
         "@nbytes",
         "@simdutf",
-        "@zlib",
```

Comment:
What's the reason for removing this? Does kj-gzip cover it?

Reply:
Yes, this also adds lots of warnings to edgeworker.